src/utils/injected.ts

```diff
@@ -59,12 +59,19 @@ export function createCoinWithAnimation() {
   }, 100);
 }
 
+var messageId = 0;
+
 export const callAPI = (message: Omit<MessageFormat, "ext" | "sender">) =>
   new Promise<void | any>((resolve, reject) => {
+    // give every message a unique autoincrementing id
+    let id = messageId;
```

Comment:
We won't change this, let's use a `const` here

Reply:
good idea